Gigamon_HAWK/GigamonHawkApplicationMetadataIntelligence.json

```diff
@@ -17,7 +17,7 @@
                 "type": "BeginBoundedTimeRange",
                 "from": {
                     "type": "LiteralTimeRangeBoundary",
```

Comment:
Did you test the dashboards after the change, I think LiteralTimeRangeBoundary won't work with relative time range.

Reply:
Yeah, this won't work. Need to update this.